tests/cli/cli-parser-test.php

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env php
+<?php
+declare(strict_types=1);
+
+require(__DIR__ . '/../../vendor/autoload.php');
+require(__DIR__ . '/CommandLineParserTest.php');
+
+$static_method = getenv('CLI_PARSER_TEST_STATIC_METHOD');
+
+// @phpstan-ignore-next-line
```

Comment:
Alternative welcome, if possible

Reply:
I can't see a good alternative to this as things currently stand. If we moved away from using `getopt()` we could eliminate the `cli-parser-test.php` file altogether and so simplify `CommandLineParserTest` a fair bit. Until then I think we may be stuck with it.

Reply:
Fixed in https://github.com/KasimirCash/FreshRSS/pull/1